modular_zubbers/code/modules/radstorm_proof_dorms/radiation_storm.dm

```diff
@@ -0,0 +1,5 @@
+/datum/weather/rad_storm/New()
+	. = ..()
+
+	for (var/area/station/commons/dorms/dorm_subtype in subtypesof(/area/station/commons/dorms))
+		protected_areas += dorm_subtype
```

Comment:
Move this to `modular_zubbers/code/datums/weather/weather_types/radiation_storm.dm`
Bonus points for also moving the code in `modular_zubbers/maps/offstation/dauntless/areas.dm#L318` there